package.json

```diff
@@ -22,54 +22,50 @@
     "prepare": "husky install"
   },
   "dependencies": {
-    "@headlessui/react": "^1.4.0",
-    "@heroicons/react": "^1.0.4",
-    "@svgr/webpack": "^5.5.0",
-    "classnames": "^2.3.1",
-    "gatsby": "^3.12.0",
-    "gatsby-plugin-image": "^1.14.0",
-    "gatsby-plugin-sitemap": "^4.8.0",
-    "normalize.css": "^8.0.1",
+    "clsx": "^1.1.1",
+    "gatsby": "^3.14.5",
+    "gatsby-plugin-image": "^1.14.1",
     "prop-types": "^15.7.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-helmet": "^6.1.0",
-    "sass": "^1.38.1"
+    "tailwindcss": "^2.2.19",
+    "tailwindcss-safe-area": "^0.2.1"
```

Comment:
Let's add in to do, a give a good explanation of why it's important to use safe-are, with good examples etc, so we can throw an issue in official tailwind repo. If it's a really must have thing for every website, it should be in official tailwind build.

Reply:
Not sure about an issue there already should an issue for this if there is already a plugin for this

Reply:
We will decided it later, lets just keep it in mind.